test/runtests.jl

```diff
@@ -66,6 +66,17 @@ using FixedSizeArrays
         end
     end
 
+    @testset "`map`" begin
+        for dim_count ∈ 0:3
+            size = ntuple(Returns(3), dim_count)
+            a = FixedSizeArray{Int, dim_count}(undef, size)
+            for v ∈ (3, 3.1, nothing)
+                @test all(==(v), map(Returns(v), a))
+                @test map(Returns(v), a) isa FixedSizeArray{typeof(v), dim_count}
```

Comment:
Not greatly important, but could avoid doing the `map` twice
```suggestion
                mapped = map(Returns(v), a)
                @test all(==(v), mapped)
                @test mapped isa FixedSizeArray{typeof(v), dim_count}
```